crates/env/src/types.rs

```diff
@@ -211,22 +211,19 @@ pub type RentFraction = Perbill;
 /// This is a mirror of the `AccountId` type used in the default configuration
 /// of PALLET contracts.
 #[derive(
-    Debug,
-    Copy,
-    Clone,
-    PartialEq,
-    Eq,
-    Ord,
-    PartialOrd,
-    Hash,
-    Encode,
-    Decode,
-    From,
-    Default,
+    Debug, Copy, Clone, PartialEq, Eq, Ord, PartialOrd, Hash, Encode, From, Default,
 )]
 #[cfg_attr(feature = "std", derive(TypeInfo))]
 pub struct AccountId([u8; 32]);
 
+impl Decode for AccountId {
+    fn decode<I: scale::Input>(input: &mut I) -> Result<Self, scale::Error> {
```

Comment:
what happens to the benchmarks if you add #[inline] here?

Reply:
I didn't get any effect=) If you want I can try to put it here, but how we will see the impact?

Funny moment, with this PR https://github.com/paritytech/ink/pull/1017, this part of code requires `#[inline(never)]`.
Without inline `Erc20` takes 9.2KB, with `#[inline(never)]` it takes 8.8KB